faq/deploy-and-maintain-faq.md

```diff
@@ -245,6 +245,13 @@ Two reasons:
 - The two results are calculated in different ways. `information_schema.tables.data_length` is an estimated value by calculating the averaged length of each row, while the store size on the TiKV monitoring panel sums up the length of the data files (the SST files of RocksDB) in a single TiKV instance.
 - `information_schema.tables.data_length` is a logical value, while the store size is a physical value. The redundant data generated by multiple versions of the transaction is not included in the logical value, while the redundant data is compressed by TiKV in the physical value.
 
+#### Why does the transaction not use the Async Commit or the one-phase commit feature?
+
+In the following situations, even you have enabled the [Async Commit](/system-variables.md#tidb_enable_async_commit-new-in-v50) feature and the [one-phase commit](/system-variables.md#tidb_enable_1pc-new-in-v50) feature using the system variables, TiDB will not use these features:
+
+- If you have enabled TiDB Binlog, restricted by the implementation of TiDB Binlog, TiDB does not use the Async Commit or one-phase commit feature.
+- Only when TiDB writes no more than 256 Key-value pairs in the transaction and the total size of keys is no more than 4KB, TiDB will have Async Commit or one-phase commit enabled to use the features. This is because for the transaction where the amount of data to be written is large, Async Commit is unable to greatly improve performance.
```

Comment:
```suggestion
- TiDB uses the Async Commit or one-phase commit features only when no more than 256 key-value pairs are written in the transaction and the total size of keys is no more than 4 KB. This is because, for transactions with a large amount of data to write, using Async Commit cannot greatly improve the performance.
```